lib/service/service.go

```diff
@@ -133,18 +133,10 @@ func (process *TeleportProcess) connectToAuthService(role teleport.Role) (*Conne
 	storage := utils.NewFileAddrStorage(
 		filepath.Join(process.Config.DataDir, "authservers.json"))
 
-	var authServers []utils.NetAddr
-	authServers, err = storage.GetAddresses()
-	if err != nil && len(authServers) == 0 {
-		log.Infof("no auth servers are available from the local storage")
-		authServers = process.Config.AuthServers
-	}
-
-	log.Infof("connecting to auth servers: %v", authServers)
 	authUser := identity.Cert.ValidPrincipals[0]
 	authClient, err := auth.NewTunClient(
 		string(role),
-		authServers,
+		process.Config.AuthServers,
```

Comment:
yeah, look at `NewTunClient()` implementation if you're curious. it checks its `addrStorage` and reads auth servers from there.